UPGRADE.md

```diff
@@ -94,9 +136,9 @@ One of the biggest complaints of users of Ilios 2.x, regarding the application,
 
 We have provided a solution to this problem with a fix that will allow you to quickly remove all but only the allowed characters from your item title and description text.  To apply this fix and reformat all your text to only use the proper formatting native to Ilios 3, do the following:
 
-* Navigate to your Ilios application root directory ('/web/ilios3/ilios' for example) and run the following command for each respective text-type:
+Navigate to your Ilios application root directory (`/web/ilios3/ilios`, for example) and run the following command for each respective text-type:
 
-```bash
+ ```bash
```

Comment:
There should be not space before the first tick-mark (`), otherwise markdown doesn't render the syntax...